### ISSUE TITLE
Suggestion: Fluid Terminus Pipe

### ISSUE THREAD

**Issue by dominic-jones** [RS485/LogisticsPipes#1519]
Usecase: Wanting to void excess fluids entering a system. A low priority route, similar to an (item) Terminus Module would be helpful. This could be linked to a fluid trashcan of sorts.

However, I can't see many other uses for this type of pipe. To be useful in the voiding of fluids, it would need multiple filters. That breaks the mould of other Fluid pipes having only one filter, at which point you might as well be explicit and call it a Fluid Void Pipe.

A potential other use case would be instead of voiding, sending excess storage of, for instance, a burnable to your furnace line. I'd use that with my present supply of Methane for instance.

**Comment by desagas** [comment]
I agree. I am playing with GregTech right now, trying to automate, and with enderio ducts causing extreme amounts of unnecessary FPS lag, I switched to LP, but only found out after rebuilding that they do not have any way to prioritize extra or excess fluids.

**Comment by dominic-jones** [comment]
I've actually wanted a tank with the ability to void excess (like barrels/drawers do) for a while now. My solution here in the end was to add a syphon to the tops of my tanks to drain off the excess if it's too full. It was a clunk solution to add to several dozen fluid types though.

LP's support for fluids has been first-class though. Most of the AE2 equivalents have been clunky in comparison.

**Comment by desagas** [comment]
I was thinking about that the other day as well. Having a void excess barrel would be perfect. I have no switched back to Super Factory Manager, as it was finally fixed and updated to no mess up inventory mapping.

I really hope that AE eventually adds the support for crafting with and for fluids. Like LP, where you can say insert 4 iron plates and some lubricant and get out a gear, or whatever. Right now, it is extremely limited. Even some addon mod for it would be amazing that added this functionality!

**Comment by dominic-jones** [comment]
There are several addon mods that do that for AE2. Both Extra Cells, and Volumetric Flask add craft with x amount of liquid. If you more examples, we can take that discussion away from this thread.

However nothing has the fine control of LP's supply level management with fluids.
